### ISSUE TITLE
Ruby Alpine 3.9 docker image is not 3.9.x with x>0

### ISSUE THREAD

**Issue by flori** [docker-library/ruby#272]
I noticed that the ruby docker image for Alpine depends on Alpine 3.9 and isn't rebuilt for later point releases like 3.9.1, 3.9.2,… but stays on 3.9.0 which was current back then. Is there a way to make docker hub rebuild an new ruby docker image once it notices that there is a new point release of its base image?

**Comment by wglambert** [comment]
What do you get if you `docker pull` it?

```console
$ docker pull ruby:alpine
alpine: Pulling from library/ruby
Digest: sha256:7fdd3e9ab2f9797c280deeb1a97d8158b28bd33eb98531b928cb006270712d2e
Status: Image is up to date for ruby:alpine

$ docker run --rm ruby:alpine cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.9.2
PRETTY_NAME="Alpine Linux v3.9"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://bugs.alpinelinux.org/"
```

**Comment by flori** [comment]
The problem was that if I have already built an image  based on ruby with alpine 3.9.0 it doesn't pull alpine 3.9.x (x> 0) again and uses the already existing 3.9.0 even after pulling the ruby docker image.

I just tried it and if I manually pull alpine:3.9 first it fetches the newest alpine:3.9.2. So what works is

```
docker pull alpine:3.9 # pulls 3.9.2 at the moment
docker pull ruby:2.6.1-alpine3.9 # Pulling ruby docker image
docker build . # building my image with FROM ruby:2.6.1-alpine3.9 based on alpine 3.9.2
```

I was expecting that pulling the ruby:2.6.1-alpine3.9 docker image would recursively pull all of its dependencies, but apparently that is not the case. Maybe it's also not a good idea to automatically pull newer versions because it could eventually break things. I think I will just pull the point release of the alpine image explicitly before building. Thx!
